### ISSUE TITLE
wee styling updates on search form

### ISSUE THREAD

**Issue by ynnig** [ror-community/ror-site#15]
The mint-green text is failing WCAG tests (accessibility guidelines) as was the brand's orange colour so we've updated that orange to a darker one and applied that to all links and accents across the rest of the site. Secondly, I also noticed the icon next to each ROR ID uses the previous, more blue, icon/logo colour. So, please can you:

1) Update all the green links/text to be dark orange in keeping with the refresh of main site pages #DA3200
2) Update the icons next to each ID to be this one (more greenish teeth)
![ror-favicon-rgb](https://user-images.githubusercontent.com/6958740/61628291-cd7c3380-ac79-11e9-9d89-466c3fa6dfc9.png)

Thanks!

**Comment by ynnig** [comment]
Or point search at the new style sheet?
